packages/bpk-docs/src/pages/BadgePage/BadgePage.js

```diff
@@ -136,42 +135,38 @@ const components = [
     blurb: [
       <Paragraph>The light badge is a gray color as shown below.</Paragraph>,
     ],
-    examples: [
-      <BadgeLayout>
-        <BpkBadge type={BADGE_TYPES.light} className={badgeClassName}>
-          Apples
-        </BpkBadge>
-        <BpkBadge type={BADGE_TYPES.light} className={badgeClassName}>
-          Bananas
-        </BpkBadge>
-        <BpkBadge type={BADGE_TYPES.light} className={badgeClassName}>
-          Strawberries
-        </BpkBadge>
-        <BpkBadge type={BADGE_TYPES.light} className={badgeClassName}>
-          Pears
-        </BpkBadge>
-      </BadgeLayout>,
+    examplesDark: [
```

Comment:
I'm not really sure about this. Wouldn't you think a new property, like `dark: true` would be better?